Group1:
  process_angle : 1.57079632679
  process_offset : 2.0
  process_amplitude : 0.1